localstack/utils/aws/aws_stack.py

```diff
@@ -239,6 +244,11 @@ def connect_to_resource_external(
     """
     Generic method to obtain an AWS service resource using boto3, based on environment, region, or custom endpoint_url.
     """
+    warnings.warn(
+        "`connect_to_resource_external` is obsolete. Use `localstack.aws.connect`",
+        DeprecationWarning,
+    )
+
```

Comment:
With this PR, these utilities are no longer used in the codebase but I've added these warnings so that they're properly highlighted in the IDE.

Btw, Python 3.12 will have a decorator to do exactly this https://peps.python.org/pep-0702/